lib/matplotlib/backends/backend_svg.py

```diff
@@ -901,8 +901,10 @@ def _draw_text_as_path(self, gc, x, y, s, prop, angle, ismath, mtext=None):
         style = {}
         if color != '#000000':
             style['fill'] = color
-        if gc.get_alpha() != 1.0:
+        if gc.get_forced_alpha() and gc.get_alpha() != 1.0:
```

Comment:
I would just remove the check on gc.get_alpha() here and have a straight if else... no?

Reply:
I put that in because a similar check appears at line 421, and it sort of makes it clearer that alpha should only be set if explicitly told to do so. Should it be changed?

Reply:
OK so we don't want to write the opacity if not needed (reasonable).  I think here the flow is clearer with
```
alpha = gc.get_alpha() if gc.get_forced_alpha() else gc.get_rgb()[3]
if alpha != 1:
    style["opacity"] = short_float_fmt(alpha)
```

Reply:
Yep, that's cleaner. Made the fix.